pypixplore/local.py

```diff
@@ -16,3 +17,17 @@ def show(self, name=None):
 
     def upgradeable(self):
         raise NotImplementedError
+
+    def package_status(self, package_name):
+        """
+        Check whether package_name is installed. If so, returns its version
+        :param package_name: str to be consulted by function
+        :return:  if installed - returns tuple with name of package and version
+                  if not installed - returns 0
+        """
+        pack_list = self.installed
```

Comment:
No need to replicate the variable here. You can iterate directly over self.installed